plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/importapp/ImportApplicationWizard.java

```diff
@@ -44,6 +44,32 @@ public ImportApplicationWizard() {
 	}
 
 	@Override
+	public void init(IWorkbench workbench, IStructuredSelection selection) {
+		Connection connection = UIUtils.getFirstElement(selection, Connection.class);
+		if (connection != null) {
+			model.setConnection(connection);
+		} else {
+			IResource resource = UIUtils.getFirstElement(selection, IResource.class);
```

Comment:
@adietish hmm...if  IProject#getProject returns null, this is a bug in the restclient IMO and we should fix it there